### PR TITLE
Converted rest of the links to GitHub

### PR DESCRIPTION
**Pull request by parthnagdev**
Converted all the possible links for which I could find a GitHub repo. Some projects in the list are outdated, some are abandoned, for such projects I did replace the link if I could find one. Some links in the list don't exist/don't work.
Let me know of all the errors if there are any and thanks, I enjoyed doing my first contributions ever! If you have more "Good first Issue" then please do let me know!

### CODE REVIEW

File: src/public/LIST.md [hunk 1]
```diff
@@ -570,18 +570,18 @@ _Tools that help send messages between clients to ensure protocol independency._
...
 - [Eureka](https://github.com/Netflix/eureka) - REST-based service registry for resilient load balancing and failover.
-- [Helidon](https://helidon.io) - Two-style approach for writing microservices: Functional-reactive and as an implementation of MicroProfile.
+- [Helidon](https://github.com/helidon-io/helidon) - Two-style approach for writing microservices: Functional-reactive and as an implementation of MicroProfile.
 - [JDA](https://github.com/discord-jda/JDA) - Wrapping of the Discord REST API and its WebSocket events.
 - [kubernetes-client](https://github.com/fabric8io/kubernetes-client) - Client provides access to the full Kubernetes & OpenShift REST APIs via a fluent DSL.
-- [Micronaut](https://micronaut.io) - Modern full-stack framework with focus on modularity, minimal memory footprint and startup time.
-- [Nacos](https://nacos.io) - Dynamic service discovery, configuration and service management platform for building cloud native applications.
+- [Micronaut](https://github.com/micronaut-projects) - Modern full-stack framework with focus on modularity, minimal memory footprint and startup time.
```
Comment (ResamVi) [inline src/public/LIST.md]: Can you use https://github.com/micronaut-projects/micronaut-core for this one please?

File: src/public/LIST.md [hunk 2]
```diff
@@ -753,64 +753,64 @@ _Frameworks that are suites of multiple libraries encompassing several categorie
 
 #### Apache Commons
```
Comment (ResamVi) [inline src/public/LIST.md]: Not your mistake but can you remove one `#` here?

So its `### Apache Commons`

**Comment by parthnagdev** [comment]
done, also am i doing this right? Something tells me there shouldn't be these many commits idk hmm i may be partially lost but hopefully it works out.

**Comment by ResamVi** [comment]
A commit should suffice without merging. But you needn't worry about the amount of commits they will get squashed into a single commit anyway.

Looks good for now, I'll have to do some tweaks afterwards but tyvm!

**Pull request merged**